hbase/src/test/java/org/apache/metamodel/hbase/CreateTableTest.java

```diff
@@ -0,0 +1,209 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.metamodel.hbase;
+
+import static org.junit.Assert.*;
+
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.apache.metamodel.MetaModelException;
+import org.apache.metamodel.schema.ImmutableSchema;
+import org.junit.Test;
+
+public class CreateTableTest extends HBaseUpdateCallbackTest {
+
+    /**
+     * Check if creating table is supported
+     */
+    @Test
+    public void testCreateTableSupported() {
+        assertTrue(getUpdateCallback().isCreateTableSupported());
+    }
+
+    /**
+     * Create a table with an immutableSchema, should throw a IllegalArgumentException
+     */
+    @Test
+    public void testWrongSchema() {
+        final ImmutableSchema immutableSchema = new ImmutableSchema(getSchema());
+        try {
+            getUpdateCallback().createTable(immutableSchema, TABLE_NAME).execute();
+            fail("Should get an exception that the schema isn't mutable");
```

Comment:
There is a different manner to test for exceptions in JUnit4 using rules. 

You can add this `rule` as a field member to the class:
```
    @Rule
    public ExpectedException exception = ExpectedException.none();
```

Then here in this method, before executing the createTable, you then add:

```
        exception.expect(IllegalArgumentException.class);
        exception.expectMessage("Not a mutable schema: " + immutableSchema);
```

For more info on this go to https://github.com/junit-team/junit4/wiki/exception-testing (and http://baddotrobot.com/blog/2012/03/27/expecting-exception-with-junit-rule/index.html).